<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1475</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>温湿度传感器
</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二氧化碳传感器</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BroadCast_pushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>Broadcast</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SSID_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PWD_lineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Show_textEdit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>210</y>
     <width>461</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="SSID_checkBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>260</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="PWD_checkBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>320</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>260</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>WIFI名字：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>WIFI密码：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
